### PR TITLE
docs(storybook): add areachart.stories.tsx

### PR DESCRIPTION
**Pull request by proke03**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: storybook/stories/chart/areachart/AreaChart.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  component: AreaChart,
+} as ComponentMeta<typeof AreaChart>;
+
+const data: TimeSeriesData[] = [];
+for (let num = 30; num >= 0; num--) {
+  data.push({
+    date: subDays(new Date(), num).toISOString().substring(0, 10),
+    value: 1 + Math.random(),
+  });
+}
```
Comment (nikolasrieble) [inline storybook/stories/chart/areachart/AreaChart.stories.tsx:18]: Lets reuse the existing time series data.

File: storybook/stories/chart/areachart/AreaChart.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            }}
+          />
+
+          <YAxis
+            dataKey="value"
+            axisLine={false}
+            tickLine={false}
+            tickCount={8}
+            tickFormatter={(number) => `$${number.toFixed(2)}`}
+          />
```
Comment (nikolasrieble) [inline storybook/stories/chart/areachart/AreaChart.stories.tsx:53]: A story about the AreaChart should not show any complicated behaviour of other components, but focus only on the AreaChart.

These are great examples, but they should be documented in the YAxis and XAxis stories.
```suggestion
```

File: storybook/stories/chart/areachart/AreaChart.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  render: (args: any) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <AreaChart data={data}>
+          <defs>
+            <linearGradient id="color" x1="0" y1="0" x2="0" y2="1">
+              <stop offset="0%" stopColor="#2451B7" stopOpacity={0.4} />
+              <stop offset="75%" stopColor="#2451B7" stopOpacity={0.05} />
+            </linearGradient>
+          </defs>
```
Comment (nikolasrieble) [inline storybook/stories/chart/areachart/AreaChart.stories.tsx:30]: Could you please split this one story up into separate ones, each showing some feature? 

(1) Most simple
(2) with Gradient
(3) ... any other ideas you have.

File: package.json [hunk 4]
```diff
@@ -61,6 +61,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "date-fns": "^2.29.3",
```
Comment (nikolasrieble) [inline package.json:64]: Lets not add storybook dependencies to the project dependencies.

**Review by nikolasrieble — CHANGES_REQUESTED** [review]
Please do not add a new dependency to the project. Also please consider my suggestions.

File: package.json (hunk 4 above)
Comment (ckifer) [reply package.json:64]: +1 this adds to bundle size which we don't need

**Review by ckifer — APPROVED** [review]

**Pull request merged**